packages/web-app-files/src/components/SideBar/Shares/FileLinks.vue

```diff
@@ -405,15 +411,15 @@ export default defineComponent({
             passwordPolicyService: this.passwordPolicyService
           },
           (newPassword) => {
-            this.createLink({ params: { ...paramsToCreate, password: newPassword }, onError })
+            this.createLink({ params: { ...paramsToCreate, password: newPassword } })
           }
         )
       } else {
-        this.createLink({ params: paramsToCreate, onError })
+        this.createLink({ params: paramsToCreate })
       }
     },
 
-    checkLinkToUpdate({ link, onSuccess = () => {} }) {
```

Comment:
I removed all those onSuccess and onError callbacks because they actually weren't used and added to confusion.